src/app/staticanalysis/modules/functionCallGraph.js

```diff
@@ -16,6 +16,30 @@ function buildLocalFuncCallGraphInternal (functions, nodeFilter, extractNodeIden
   return callGraph
 }
 
+/**
+ * Builds a function call graph for the current contracts.
+ * Example Contract call graph:
+ *
+ * {
+ *  "KingOfTheEtherThrone": {
+ *    "contracts": {...},                                        // Contract node as defined in abstractAstView.js
+ *    "functions": {
+ *      "KingOfTheEtherThrone.claimThrone(string memory)": {    // function in KingOfEtherThrone
```

Comment:
I also tried it via import. I noticed that in `compilationResult.sources` the files are not ordered in source order (imports first, then the current file). At the moment the static analysis modules rely on that if you inherit from a contract it is defined before some other contract inherits form it (if you only use one file this is actually enforced by the compiler). Is there an easy way to change the order in the source array such it reflects source order?